Source/Managers/LuaMan.cpp

```diff
@@ -923,40 +941,116 @@ namespace RTE {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-	const std::vector<std::string> * LuaMan::DirectoryList(const std::string &filePath) {
+	// TODO: Ask Causeless whether this is an appropriate spot for this method
+	std::string LuaMan::GetCaseInsensitiveFullPath(const std::string &fullPath) {
```

Comment:
@Causeless Thoughts on this TODO?

Reply:
Move it to RTETools - it can go alongside StringsEqualCaseInsensitive()

Reply:
Done